backend/db/crud_jmp.py

```diff
@@ -0,0 +1,83 @@
+from sqlalchemy.orm import Session
+from typing import List
+from AkvoResponseGrouper.views import get_categories
+from models.data import DataDict, Data
+
+# Workaround while WAI to AkvoResponseGrouper migration is in progress
+config_items = [
```

Comment:
Please remove this config items @ifirmawan

Reply:
Okay mas